modules/core_api/fsw/inc/cfe_version.h

```diff
@@ -26,8 +26,9 @@
 #define CFE_VERSION_H
 
 /* Development Build Macro Definitions */
-#define CFE_BUILD_NUMBER   405 /**< @brief Development: Number of development git commits since CFE_BUILD_BASELINE */
-#define CFE_BUILD_BASELINE "v7.0.0-rc4" /**< @brief Development: Reference git tag for build number */
+#define CFE_BUILD_NUMBER   	405 /**< @brief Development: Number of development git commits since CFE_BUILD_BASELINE */
+#define CFE_BUILD_BASELINE 	"equuleus-rc1" /**< @brief Development: Reference git tag for build number */
+#define CFE_BUILD_DEV_CYCLE 	"equuleus-rc2" /**< @brief Development: Release name for current development cycle */
 
 /* See \ref cfsversions for definitions */
```

Comment:
Consider unique MAJOR.MINOR.REVISION.MISSION_REV per update or at minimum per RC, since this is what comes down in tlm.  I was a fan of using incrementing odd numbers to indicate development versions, at a level that matches the impact of changes to date since last official release?  Note repeated/copied string in multiple places adds to maintenance overhead of version changes, consider a macro or helper function?  Or just use the CFE_CONFIGID_CORE_VERSION_DESCRIPTION?

Reply:
CCB 11 January 2024: Will have unique major/minor/revision. Build number provides uniqueness per update.